nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -234,6 +234,10 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
   To be able to play DRM protected content in `chromium` now, you have to explicitly opt-in as originally intended using `chromium.override { enableWideVine = true; }`.
   This override was added almost 10 years ago.
 
+- `clang-tools_<version>` packages have been moved into `llvmPackages_<version>` (i.e. `clang-tools_18` is now `llvmPackages_18.clang-tools`).
+  - For convenience, the top-level `clang-tools` attribute remains and is now bound to `llvmPackages.clang-tools`.
+  - Top-level `clang_tools_<version>` attributes are now aliases; these will be removed in a future release.
+
```

Comment:
Since we're not backporting this PR we should move this to the [24.11 release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md#backward-incompatibilities-sec-release-2411-incompatibilities).

Reply:
Addressed.